src/new/wallet.js

```diff
@@ -515,7 +522,7 @@ class HathorWallet extends EventEmitter {
           sigs.push(hexToBuffer(p2shSig.signatures[index]));
         } catch (e) {
           // skip if there is no signature, or if it's not hex
-          continue
+          continue;
```

Comment:
note(non-blocking): Codecov is raising one case that was not tested: having invalid p2sh signatures.

Since P2SH transactions involve message exchange and direct user input I think it's something actually _plausible_ to happen. Maybe we could open an issue to remind us of this test case.

Reply:
Can you create an issue for this?
I think its important, but it is beyond the scope of this PR.

Reply:
Opened #618 requesting this test